modules/standard/CommDiagnostics.chpl

```diff
@@ -351,9 +351,9 @@ module CommDiagnostics
       D(loc.id).test_nb = chpl_numCommTestNB();
       D(loc.id).wait_nb = chpl_numCommWaitNB();
       D(loc.id).try_nb = chpl_numCommTryNB();
-      D(loc.id).fork = chpl_numCommForks();
-      D(loc.id).fork_fast = chpl_numCommFastForks();
-      D(loc.id).fork_nb = chpl_numCommNBForks();
+      D(loc.id).execute_on = chpl_numCommForks();
```

Comment:
I removed them and it seems to be working... so, fingers crossed, I'll test again.